core/src/plugins/filed/python/postgresql/bareos-fd-postgresql.py

```diff
@@ -888,26 +999,11 @@ def __wait_for_wal_archiving(self, lsn):
         for the given LSN is present in the filesystem.
         """
         bareosfd.DebugMessage(100, "__wait_for_wal_archiving() started\n")
-        if self.pg_major_version >= 10:
-            wal_filename_func = "pg_walfile_name"
-        else:
-            wal_filename_func = "pg_xlogfile_name"
+        # should be already done.
+        if not self.wal_filename:
+            self.__decode_lsn_filename_offset(lsn)
```

Comment:
why are we ignoring the return value ?

Reply:
Let me recheck that point with all other modification in.